packages/components/src/components/Divider/ContentfulDivider.tsx

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
 import './ContentfulDivider.css';
-import { css, cx } from 'emotion';
-
-const styles = {
-  hr: css({
-    border: 'none',
-  }),
-};
 
 export type ContentfulDividerProps = {
   className?: string;
 };
 
-export const ContentfulDivider = (props: ContentfulDividerProps) => {
-  const { className } = props;
+export const ContentfulDivider = ({ className = '', ...props }: ContentfulDividerProps) => {
```

Comment:
This subtle change makes it so `className` will not be spread via `...props` to the `div.cf-divider` element